app/scripts/modules/kubernetes/src/v2/manifest/yaml/yamlEditor.less

```diff
@@ -1,6 +1,6 @@
 #yaml-editor {
```

Comment:
maybe this file could be renamed and the selectors could apply to both editors?

Reply:
Done!